spring-cloud-commons/src/test/java/org/springframework/cloud/configuration/SpringBootDependencyTests.java

```diff
@@ -204,15 +205,15 @@ private void verifyCurrentVersionFromManifest(String version) {
 
 	@Test
 	public void should_match_against_current_predicate() {
-		List<String> acceptedVersions = Collections.singletonList("3.0");
```

Comment:
I didn't think this one needs to change

Reply:
If we don't change it, we'll get:
```
java.lang.AssertionError: 
Expecting empty but was: "Spring Boot is not compatible with this Spring Cloud release train"
```
in the test.